phpunit/functional/TaskTemplateTest.php

```diff
@@ -46,4 +46,129 @@ protected function getInstance(): \AbstractITILChildTemplate
     {
         return new \TaskTemplate();
     }
+
+    public function testCurrentUserFeature()
+    {
+        $this->login();
+        $template = new \TaskTemplate();
+
+        $template_id = $template->add([
+            'name'         => 'Test Current User Template',
+            'content'      => 'Test content',
+            'users_id_tech' => -1
+        ]);
+        $this->assertGreaterThan(0, $template_id);
```

Comment:
This applies to the other add() and update() tests.

Use createItem() or updateItem(), it's easier to read and more robust.
```suggestion
        $template_id = $this->createItem('TaskTemplate', [
            'name'         => 'Test Current User Template',
            'content'      => 'Test content',
            'users_id_tech' => -1
        ])->getID();
```